teammapper-frontend/src/assets/i18n/de.json

```diff
@@ -167,7 +167,8 @@
     "NODE_COPIED": "Knoten erfolgreich kopiert!",
     "ERRORS": {
       "ROOT_NODE_COPIED": "Es ist nicht möglich, den Hauptknoten zu kopieren!",
-      "GENERIC_NODE_COPY_ERROR": "Beim Kopieren des Knotens ist ein Fehler aufgetreten."
+      "GENERIC_NODE_COPY_ERROR": "Beim Kopieren des Knotens ist ein Fehler aufgetreten.",
```

Comment:
I would suggest to use a clearer prefixing of translations, e.g.:

`NODE_COPY_GENERIC_ERROR` start with the most important information first and not last. ERROR at the end here is fine or could be removed as you already have it under this key. Otherwise I would put it first, e.g. `ERROR_NODE_COPY_GENERIC`

Reply:
This is now in a separate PR: https://github.com/b310-digital/teammapper/pull/330